USAGE.md

```diff
@@ -45,9 +46,9 @@ See the [Hue Emulation](https://www.openhab.org/addons/integrations/hueemulation
     * The interactive REST API interface available through the openHAB 2 dashboard
     * PaperUI does not yet allow manipulation of tags
 
-#### Item Label Recommendation
+#### Item Label Recommendation and Requirement
 
-Matching of voice commands to Items happens based on the Item label (e.g. "Kitchen Light").
+Matching of voice commands to Items happens based on the Item label (e.g. "Kitchen Light").  (Note: Although, it is valid syntax to not have an item label, the string in quotes following the name.  For this skill to function properly a label is required.)
```

Comment:
This seems like an odd place to talk about requirements, can we state labels are required in the section above, where we also talk about needing a tag?